include/circt/Conversion/Passes.td

```diff
@@ -37,6 +37,40 @@ def AffineToPipeline : Pass<"convert-affine-to-pipeline", "mlir::func::FuncOp">
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// ExportChiselInterface and ExportSplitChiselInterface
+//===----------------------------------------------------------------------===//
+
+def ExportChiselInterface : Pass<"export-chisel-interface", "firrtl::CircuitOp"> {
+  let summary = "Emit a Chisel interface to a FIRRTL circuit";
+  let description = [{
+    This pass generates a Scala Chisel interface for the top level module of
+    a FIRRTL circuit.
+  }];
+
+  let constructor = "createExportChiselInterfacePass()";
+  let dependentDialects = [
+    "circt::firrtl::FIRRTLDialect"
+  ];
+}
+
+def ExportSplitChiselInterface : Pass<"export-split-chisel-interface", "firrtl::CircuitOp"> {
```

Comment:
I went with the `Export/ExportSplit...` convention used for verilog and systemc emission even though only one file is exported for now.